test/multiprocessing_test_case.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+import multiprocessing
+import sys
+import tempfile
+import time
+import unittest
+from collections import namedtuple
+from functools import wraps
+
+import torch
+import torch.distributed as c10d
+from common_utils import TestCase, NO_MULTIPROCESSING_SPAWN
+
+
+TestSkip = namedtuple("TestSkip", "exit_code, message")
+
+
+TEST_SKIPS = {
+    "multi-gpu": TestSkip(75, "Need at least 2 CUDA devices"),
+    "nccl": TestSkip(76, "c10d not compiled with NCCL support"),
+    "known_issues": TestSkip(77, "Test skipped due to known issues"),
+}
+
+
+def skip_if_not_multigpu(func):
```

Comment:
Can we import the one from `common_distributed`, or is this one different than that?